### PR TITLE
Ram: ProMaster support

### PR DESCRIPTION
**Pull request by jande48**
Hi Castle Hill, 
   Below is a PR to include the new Promaster firmware versions and fingerprint.

### CODE REVIEW

File: selfdrive/car/chrysler/values.py [hunk 1]
```diff
@@ -130,6 +134,7 @@ class ChryslerCarInfo(CarInfo):
...
   }],
+  CAR.PROMASTER: [{218: 8, 220: 8, 222: 6, 224: 4, 238: 8, 241: 8, 250: 8, 252: 8, 254: 8, 257: 8, 262: 7, 270: 7, 271: 8, 278: 8, 279: 4, 346: 4, 498: 8, 501: 5, 502: 4, 506: 8, 552: 8, 554: 8, 557: 4, 592: 4, 597: 8, 599: 8, 601: 8, 748: 8, 762: 4, 896: 8, 900: 8, 1046: 8, 1048: 4, 1199: 8, 1201: 8, 1349: 8, 1351: 8, 1354: 8, 1438: 8, 1442: 8, 1444: 8, 1445: 8, 1446: 8, 1449: 8, 1456: 8, 1470: 8, 1504: 8, 1508: 8, 1849: 4, 1854: 4, 1865: 8, 1994: 2}],
```
Comment (sshane) [inline selfdrive/car/chrysler/values.py:137]:
```suggestion
```

can fingerprints for new cars are not required

**Comment by sshane** [comment]
@realfast is this car supportable?

**Comment by GregorKikelj** [comment]
> @realfast is this van the same platform as the trucks?

Seems to be a Fiat Ducato platform, doubt it's shared with trucks.

**Comment by realfast** [comment]
> @realfast is this van the same platform as the trucks?

This is Fiat Ducato based as @GregorKikelj pointed out.

**Comment by sshane** [comment]
@jande48 Were you doing the full port? This just adds FW versions, and it's a different platform as Gregor points out. It looks like you accidentally opened this to commaai/openpilot, not a fork?

**Comment by jande48** [comment]
Correct. Is this still an Open PR? I’ll take it down. Sorry, it’s not ready
for review

**Comment by sshane** [comment]
no problem, feel free to re-open when you start working on this

**Comment by bmacw** [comment]
I'm interested in getting OP to work with promaster, but I want to make sure I'm getting the right Promaster.  From the work that's been done so far, does anyone know if I need a promaster with LKAS to be able to control the EPS? Or would a normal 2022 model (with EPS) + ACC also be capable to execute lateral controls?

**Pull request closed without merging**